lab1.js

```diff
@@ -117,11 +133,16 @@ var tooHungryDay;
  pondering protein supplements (the first day the average dips below 4
  meals)
 */
+var sum = 0;
+for (var i = 0; i < mealsPerDay.length; i++) {
+  var avg = (sum += mealsPerDay[i]) / (i + 1);
+  if (avg < 4) {
+    var tooHungryDay = i - 2;
```

Comment:
good. And, if you want to stop executing the loop here (since you don't need to run the loop once you find your answer), you could do a break right here.